lib/async/io/ssl_endpoint.rb

```diff
@@ -99,6 +99,12 @@ def each
 		SecureEndpoint = SSLEndpoint
 		
 		class Endpoint
+			# @param args
```

Comment:
Something about using `args` bothers me slightly, but I'm not sure how it could be better. It's not very clear what they should be or how they should be used, or what the user should provide. Maybe explain they are passed to `tcp` and give an example? Can we embed some kind of `@example Endpoint.ssl(...)`?

Reply:
Super, we shall do this. I'll add it to the Description of this PR as a TODO.